src/Tool/FixUrlsCommand.php

```diff
@@ -68,11 +68,26 @@ protected function configure()
                 InputOption::VALUE_REQUIRED,
                 'Database name'
             )
+            ->addOption(
+                'promptmode',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Prompt for confirmations? (y)'
+            )
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        // Get promptMode - eg whether or not to prompt for confirmation to continue (handy for cron jobs)
+        $prompt = $input->getOption('promptmode');
+        if (!empty($prompt) && preg_match('/[1n](?:o)?/i', $prompt)) {
```

Comment:
can you explain this regex John? Presumably this would work just as well?..

```
$prompt = trim($input->getOption('promptmode'));
if (!empty($prompt) && preg_match('/^n|no$/i', $prompt)) { 
```

Looks like you're trying to support: promptmode=1 or promptmode=n or promptmode=no

Is that right?